src/Microsoft.Azure.SignalR.Common/Interfaces/IServiceConnectionManager.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.SignalR
+{
+    interface IServiceConnectionManager
+    {
+        IEnumerable<IServiceConnection> CreateServiceConnection(int count = 1);
```

Comment:
>CreateServiceConnection [](start = 40, length = 23)

naming: `Connections` for `IEnumerable`

Reply:
We decide to only support creating one connection